src/utils/changelog.ts

```diff
@@ -246,7 +247,8 @@ function formatCommit(commit: Commit): string {
 
 export async function generateChangesetFromGit(
   git: SimpleGit,
-  rev: string
+  rev: string,
+  maxLeftovers: number = MAX_LEFTOVERS
```

Comment:
This is fine as-is, but I generally prefer having a single object parameter as it helps when reading the code at the callsite of the function since you'll be able to see the object's keys. (i.e. python's args vs kwargs).